api/graphql/tests/snapshots/snap_test_reservation_units.py

```diff
@@ -1775,7 +1775,7 @@
                         'nameFi': 'test name fi',
                         'paymentProduct': {
                             'merchantPk': '3828ac38-3e26-4501-8556-ba2ea3442627',
-                            'pk': '1018cabd-d693-41c1-8ddc-dc5c08829048'
+                            'pk': 'cf9798b2-c5c1-4bdc-877a-47722dc39f9e'
```

Comment:
so this is coming from the mocked response? Its ok :innocent:

Reply:
Yep, but I updated the tests and used the old UUID in the mock just to avoid pointless changes. I also managed to remove few extra lines from the test file 🧹